### PR TITLE
Jancizmar/add project dashboard page #915

### PR DESCRIPTION
**Pull request by JanCizmar**

### CODE REVIEW

File: webapp/src/views/projects/dashboard/DashboardPreview.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            </Grid>
+          </StyledLanguagesAndActivityGridContainer>
+          <DailyActivityChart />
+        </StyledContainer>
+      )}
+    </BaseView>
+  );
+};
+
+const StyledContainer = styled(Box)`
```
Comment (stepan662) [inline webapp/src/views/projects/dashboard/DashboardPreview.tsx:45]: Could we have convention that we define this before the main component?

File: webapp/src/views/projects/dashboard/ProjectTotals.tsx [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+              {t('project_dashboard_tag_count', 'Tags')}
+            </TileDescription>
+          </TileDataItem>
+        </Tile>
+      </Tiles>
+    </>
+  );
+};
+
+const Tiles = styled(Box)`
```
Comment (stepan662) [inline webapp/src/views/projects/dashboard/ProjectTotals.tsx:118]: Missing `Styled` prefix. I don't persist on that, but it's already everywhere, so I would do it like that.

File: webapp/src/views/projects/dashboard/ProjectTotals.tsx [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+const TileDescription = styled(Typography)`
+  font-size: 18px;
+`;
+
+const TileSubDescription = styled(Typography)`
+  font-size: 13px;
+  color: #808080;
+`;
+
+const OwnerAvatar = (props: {
```
Comment (stepan662) [inline webapp/src/views/projects/dashboard/ProjectTotals.tsx:160]: I think it's nicer to define Prop types separately:
```
type Props = {
  ...
}

const OwnerAvatar: React.FC<Props> = (...) => {}
```

**Review by stepan662 — APPROVED** [review]

File: webapp/src/views/projects/dashboard/DashboardPreview.tsx (hunk 1 above)
Comment (JanCizmar) [reply webapp/src/views/projects/dashboard/DashboardPreview.tsx:45]: Is really the first thing I want to read in react component the style definitions? The convention should be more defining it after, so reader could read less and directly get to the important part.

File: webapp/src/views/projects/dashboard/ProjectTotals.tsx (hunk 2 above)
Comment (JanCizmar) [reply webapp/src/views/projects/dashboard/ProjectTotals.tsx:118]: Right.

File: webapp/src/views/projects/dashboard/ProjectTotals.tsx (hunk 3 above)
Comment (JanCizmar) [reply webapp/src/views/projects/dashboard/ProjectTotals.tsx:160]: Its more verbose, but OK

**Pull request merged**